there's no glx.pc so the check fails, but the darwin workaround also finds it on
linux too.
diff --git a/meson.build b/meson.build
index c1853f9..188305c 100644
--- a/meson.build
+++ b/meson.build
@@ -99,7 +99,7 @@ if not dep_glu.found() and host_machine.system() != 'darwin'
 endif
 
 dep_glx = dependency('glx', required: false, disabler : true)
-if not dep_glx.found() and host_machine.system() == 'darwin'
+if not dep_glx.found()
   # xquartz doesn't have a glx.pc, but it does have the header. And all the
   # symbols reside in libGL, so let's just use that.
   if cc.check_header('GL/glx.h', dependencies: dep_x11)
